src/style/base/_base.scss

```diff
@@ -1,24 +1,34 @@
 // font imports
 @font-face {
-    font-family: 'Manrope';
-    src: url("../../assets/fonts/Manrope/Manrope-VariableFont_wght.ttf") format('opentype');
+    font-family: Futura-Bold;
+    src: url("../../assets/fonts/futura/Futura Bold font.ttf") format('truetype'); 
 }
 @font-face {
-    font-family: 'Sometype Mono';
-    src: url("../../assets/fonts/Sometype-Mono-master/fonts/ttf/SometypeMono-Medium.ttf") format('opentype');
+    font-family: Futura-Medium;
+    src: url("../../assets/fonts/futura/futura medium bt.ttf") format('truetype'); 
+}
+@font-face {
+    font-family: Futura-Med-Ital;
+    src: url("../../assets/fonts/futura/Futura Medium Italic font.ttf") format('truetype'); 
 }
-@import url(https://fonts.googleapis.com/css2?family=Be+Vietnam:ital,wght@0,400;0,500;1,400;1,500&display=swap);
-// fonts
-$header-font: 'Manrope', sans-serif;
-$body-font: 'Sometype Mono', sans-serif;
 
-// colors
+// fonts
+$header-font: 'Futura-Bold', sans-serif;
+$header-font2: 'Futura-Med-Ital', sans-serif;
```

Comment:
Hmm very small change but I think if we are using dashes for naming style, then we should also have a dash before the `2` like `$header-font-2`. Not a big issue so we can also make this change in the next update.